### PR TITLE
frontend first commit

### PR DESCRIPTION
**Pull request by BingBlog**
frontend first commit:
1、modify all  Chinese annotate；
2、add pre-commit to lint the code;
3、code format yet to do

### CODE REVIEW

File: visualdl/frontend/tool/build.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/**
+ * @file build script
+ * @author abingblog@gmail.com
```
Comment (jacquesqiao) [inline visualdl/frontend/tool/build.js]: 这些方便去掉么？

File: visualdl/frontend/mock/example/mock.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/**
+ * @file mock data
```
Comment (Superjomn) [inline visualdl/frontend/mock/example/mock.js]: remove these comment

File: visualdl/frontend/tool/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+            stylus: {
+                default: {
+                    use: [rider()]
+                }
+            }
+        })
+    ]
+};
+
+// 配置模板
```
Comment (Superjomn) [inline visualdl/frontend/tool/webpack.config.js]: translate these Chinese comments

File: visualdl/frontend/tool/webpack.dev.config.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+/**
+ * @file webpack
```
Comment (Superjomn) [inline visualdl/frontend/tool/webpack.dev.config.js]: remove all authors

**Pull request closed without merging**